app/views/home_page/index.html.haml

```diff
@@ -0,0 +1,5 @@
+= render :partial => "header"
```

Comment:
You should use the new hash syntax `partial: 'header'` -> read more [here](https://github.com/rubocop-hq/ruby-style-guide#no-mixed-hash-syntaxes)